index.src.html

```diff
@@ -513,20 +514,24 @@ <h3 id="fetch-integration">Fetch Integration</h3>
 
   ISSUE: Monkey patching! Talk with Anne.
 
-  If the <a>`Clear-Site-Data`</a> header is present in an HTTP
-  <a>response</a>, then data MUST be cleared before rendering the response to
-  the user. That is, before step #9 in the current <a>main fetch</a> algorithm,
-  execute the following step:
+  If the <a>`Clear-Site-Data`</a> header is present in an HTTP <a>response</a>
+  received from the network, then data MUST be cleared before rendering the
+  response to the user. That is, after step #14 in the current
+  <a>HTTP-network fetch</a> algorithm, execute the following step:
 
-  9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+  15.  If |response|'s `credentials flag` is set, and |response|'s
+       <a>header list</a> contains a header named <a>`Clear-Site-Data`</a>, then
+       execute [[#clear-response]] of [[CLEAR-SITE-DATA]] on |response|.
```

Comment:
1.  `credentials flag` isn't a property of `response`: it's an argument to the "HTTP-network fetch" algorithm. How about:

  ```
If <var ignore>credentials flag</var> is set, and |response|'s
<a>header list</a> contains a header named
<a>`Clear-Site-Data`</a>, then execute [[#clear-response]]
on |response|.
  ```

  Note the `<var ignore>` bit, which just landed in Bikeshed, and lets us mark something up as a monkey-patched variable without it giving us annoying warning messages.

2. Please drop the `[[CLEAR-SITE-DATA]]` bit; I agree that we'll want to make that reference when we upstream this to Fetch, but for the moment, it's confusing Bikeshed's tiny brain and adding a reference to this document _to_ this document, which is odd. :)